src/notation/internal/notationinteraction.cpp

```diff
@@ -492,7 +493,11 @@ void NotationInteraction::select(const std::vector<Element*>& elements, SelectTy
     for (Element* element: elements) {
         score()->select(element, type, staffIndex);
     }
+}
 
+void NotationInteraction::select(const std::vector<Element*>& elements, SelectType type, int staffIndex)
```

Comment:
Need to preserve this for external calls to public INotationInteraction::select.